client/src/main/java/com/alibaba/nacos/client/naming/backups/FailoverReactor.java

```diff
@@ -25,16 +25,23 @@
 import com.alibaba.nacos.common.lifecycle.Closeable;
 import com.alibaba.nacos.common.utils.JacksonUtils;
 
+import com.alibaba.nacos.common.utils.StringUtils;
 import com.alibaba.nacos.common.utils.ThreadUtils;
-import org.apache.commons.lang3.StringUtils;
 
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.StringReader;
 import java.nio.charset.Charset;
-import java.util.*;
+import java.util.Map;
+import java.util.Date;
+import java.util.Calendar;
+import java.util.TimerTask;
+import java.util.HashMap;
+import java.util.Arrays;
+import java.util.List;
 import java.util.concurrent.*;
 
+
```

Comment:
remove extra blank line.

Reply:
In the next commit, I will remove the blank line.